modules/api/static/schema.yml

```diff
@@ -1831,6 +1831,8 @@ components:
           type: string
         Project:
           type: string
+        ProjectID:
```

Comment:
This isn't correct. There's supposed to be a project name (line 1832) not a projectid

https://github.com/aces/Loris/blob/main/modules/api/docs/LorisRESTAPI_v0.0.4-dev.md#user-content-30-candidate-api

Reply:
It looks like it was incorrectly added in https://github.com/aces/Loris/commit/7bd3ddde3079ea667140d2c8c1079d52b73f76fc and missed in review. It should not be in the result that's serialized, only the query for the filter to work. Ihe API isn't intended to expose internal IDs.

Reply:
I agree but I dont think we can remove it now can we?

should be removed in 0.0.4 I guess

Reply:
Why can't we remove it? It's not documented as being there in any API documentation.

Reply:
I don't know... it was still being returned and someone might be using it? You are the one who usually avoid modifying the API outside of a new release. I'm okai with wtv